terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/download/AbstractFileDownloadView.java

```diff
@@ -56,7 +56,8 @@
  * <h3>Example of bean definition file</h3>
  * 
  * <pre>
- * &lt;bean id=<strong style=color:red>&quot;sampleFileDownloadView&quot;</strong> class=&quot;org.terasoluna.gfw.web.sample.SampleFileDownloadView&quot;&gt;
+ * &lt;bean id=<strong style=
+color:red>&quot;sampleFileDownloadView&quot;</strong> class=&quot;org.terasoluna.gfw.web.sample.SampleFileDownloadView&quot;&gt;
```

Comment:
- [ ] It seems that the line breaks without "*" when formatting.
I think you should add "*".

Reply:
9586aff0d3955e6460d16f0a01874aab731eb062
I modified it according to the previous line.
https://github.com/terasolunaorg/terasoluna-gfw/blob/issues/1084_format_source/terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/download/AbstractFileDownloadView.java#L46-L48